### ISSUE TITLE
Event Hash link not correct

### ISSUE THREAD

**Issue by jnovack** [bvalosek/jsdoc3-bootstrap#1]
The following code does not link to the event properly.

```
/** @class Hurl */
function Hurl() {}

/**
 * Throw a snowball.
 *
 * @fires Hurl#snowball
 */
Hurl.prototype.snowball = function() {
    /**
     * Snowball event.
     *
     * @event Hurl#snowball
     * @type {object}
     * @property {boolean} isPacked - Indicates whether the snowball is tightly packed.
     */
    this.emit('snowball', {
        isPacked: this._snowball.isPacked
    });
};

module.exports = Hurl;
```

The generated page (`Hurl.html`) produces a Method `snowball` and an Event `snowball`.  The link under the Method AND the Event generates as `#snowball`.

The Method should generate `#snowball` (and it does, properly).  The Event should generate `#event:snowball`, which it currently does not.

_edit: added full code_

**Comment by jnovack** [comment]
fixed in 0.11.2.

thank you.

**Comment by bvalosek** [comment]
update to 0.12 for these recent fixes
